tools/lkl/lib/virtio_net.c

```diff
@@ -66,32 +66,55 @@ static inline int is_tx_queue(struct virtio_dev *dev, struct virtio_queue *queue
        return &dev->queue[TX_QUEUE_IDX] == queue;
 }
 
-static int net_enqueue(struct virtio_dev *dev, struct virtio_req *req)
+/* Inside the LKL kernel, we wrapped this buffer (which corresponds to
+ * an IP packet) with a header to make sure it got routed to the TAP
+ * device. This header is meaningless to the oustide world, so we cut
```

Comment:
Yes, I see why this sounded misleading, I've changed it to be clearer. Let me know what you think.